src/Main.elm

```diff
@@ -148,7 +149,7 @@ type alias FormField =
 
 
 type InputField
-    = ShortText String (Maybe Int)
+    = ShortText String (Maybe Int) (Maybe Bool)
```

Comment:
introduce a `multiple` attribute

Reply:
feels like we need to provide a way to indicate extra attributes, instead of appending more and more `(Maybe a)`